l2tscaffolder/helpers/git.py

```diff
@@ -128,3 +128,17 @@ def SwitchToBranch(self, branch: str) -> int:
     exit_code, _, _ = self.RunCommand(command)
 
     return exit_code
+
+  def CreateBranch(self, branch: str) -> int:
+    """Creates a git branch, switches to it and returns the exit code.
```

Comment:
Rather than have this create and switch to branch, please have it run `git branch` to just create the branch. There's already a separate function for switching to a branch